### DISCUSSION TITLE
Adding modal type preview in notifications button.

### DISCUSSION THREAD

**Discussion by Neilblaze** [community/community#67] [Other Feature Feedback, Questions, & Ideas]
Does it sound good if Github adds a dropdown modal sort of preview for Github notifications bell button? Currently, on clicking the bell icon of Github notifications, the user is redirected to the notifications page. This unnecessarily consumes a lot of time. Adding to that, it would be great if the issues can be tagged according to their type. 

Say for invite notification, it's tagged in `green` color, for mention, it's `blue` .

**Comment by anburocky3** [comment]
Yea, that would be great. Since I have to go to the notification page for every notification. Which is a kind of pain.

**Reply by Neilblaze** [reply]
Exactly!

**Reply by becca** [reply]
What specifically about going to this page is a pain?

**Reply by Neilblaze** [reply]
Sorry if I had sounded a bit rude, but I was saying if like if Github adds this feature what I proposed above, then the user experience might improve, cause for viewing a single notification, the user has to visit another page, which is completely fine but I believe a bit time-consuming.

**Comment by becca** [comment]
👋 thank you for this feedback. right now the feedback in this repository is specific to our Mobile products and Discussions. I'm curious about whether this feedback became a problem with the introduction of Discussions or is related to Discussions in some way for you?

**Reply by Neilblaze** [reply]
Oh I see, my bad then :sweat: 
I thought this discussion section was open for both Web & Mobile products of Github.

**Comment by Schoologyy** [comment]
https://www.google.com/?safe=active&ssui=on
